src/tab.ts

```diff
@@ -14,14 +15,14 @@ function getNextElement(
   elements: Element[],
   focusTrap?: Document | Element,
```

Comment:
```suggestion
  focusTrap: Document | Element,
```

This should never be undefined and if it is we'll have an error in `tab()` anyway.